packages/opc-feedback/src/opc-feedback.ts

```diff
@@ -164,36 +164,11 @@ export class OpcFeedback extends LitElement {
     return this.template;
   }
 
-  get renderIcon() {
-    if(this.beetleIcon !== "hide"){
-    const isModelOpen =
-      this._openFeedbackModal ||
-      this._openConfirmationModal ||
-      this._openInitialModal ||
-      this._openBugModal;
-    if (!isModelOpen) {
-      return html`<img
-        src=${beetleIcon}
-        width="20px"
-        height="20px"
-        id="beetleIcon"
-        alt="beetle"
-        class="pf-u-mr-xs"
-      />`;
-    }
-    return html`<ion-icon
-      name="${isModelOpen ? 'ellipsis-horizontal-outline' : 'chatbox-ellipses'}"
-      class="pf-u-font-size-xl pf-u-mr-xs"
-    >
-    </ion-icon>`;
-    }
-  }
```

Comment:
@shwaghma the issue is to fix the problem with the `beetleIcon="hide"`.

This will just remove the icon completely.

This is a reusable web component, where the adopters can choose to show/hide it depending on their use case.

Can you revert this and look into the issue where they are not able to hide the icon?